src/pages/resourceManager/TrafficModelList.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ColumnsType } from 'antd/es/table';
+
+import BaseResourceList from '@/components/resourceManager/BaseResourceList';
+import { useResourceList } from '@/hooks/resourceManager/useResourceList';
+import { useDeleteTrafficModelMutation, useListTrafficModelsQuery } from '@/services/resourceManager/trafficModelApi';
+import { LoadPatternDTO } from '@/types/resourceManager/loadPattern';
+import { sortNamespace } from '@/utils/resourceManager/sortNamespace';
+
+const TrafficModelList: React.FC = () => {
+  const { t } = useTranslation();
+  const { data, isLoading, isFetching, refetch } = useResourceList({
+    resourceKind: t('TrafficModel'),
+    listHook: useListTrafficModelsQuery,
+  });
+
+  const columns: ColumnsType<LoadPatternDTO> = [
+    {
+      title: t('Name'),
+      dataIndex: ['metadata', 'name'],
+      sorter: (a, b) => a.metadata.name.localeCompare(b.metadata.name),
+    },
+    {
+      title: t('Namespace'),
+      width: 200,
+      dataIndex: ['metadata', 'namespace'],
+      sorter: (a, b) => sortNamespace(a.metadata.namespace, b.metadata.namespace),
+      defaultSortOrder: 'ascend',
+    },
+  ];
+
+  return (
+    <BaseResourceList
+      showNamespaceSelect
+      allowClone
+      allowEdit
+      resourceKind={t('TrafficModel')}
+      resourceKindUrl="trafficModel"
+      data={data}
+      isLoading={isLoading}
+      isFetching={isFetching}
+      refetch={refetch}
+      deleteHook={useDeleteTrafficModelMutation}
+      columns={columns}
+      scroll={{ x: 850 }}
```

Comment:
<img width="288" alt="Screen Shot 2024-02-24 at 11 47 13 PM" src="https://github.com/CarnegieMellon-PlantD/PlantD-Studio/assets/32251638/e86066c0-4b9b-4faf-b534-02afb7e776c8">

I don't see a scroll of 700 set elsewhere? Am I misunderstanding the number?

Reply:
It's because every table has different columns.

All the columns have a fixed width, except the first one, which will have a flexible width. The scroll width is then the sum of the width of all the fixed-width columns, plus `500`, which can be treated as a base number.